app/src/main/java/org/pesmypetcare/mypetcare/services/medication/MedicationManagerAdapter.java

```diff
@@ -55,8 +55,8 @@ public void updateMedicationKey(User user, Pet pet, String newDate, String oldDa
         String owner = user.getUsername();
         String petName = pet.getName();
         DateTime oldDateTime = DateTime.Builder.buildFullString(oldDate);
-        MedicationData medicationData = ServiceLocator.getInstance().getMedicationManagerClient().getMedicationData(
-            accessToken, owner, petName, oldDateTime, oldName);
+
+        MedicationData medicationData = ServiceLocator.getInstance().getPetCollectionsManagerClient().getMedication(user.getToken(), user.getUsername(), pet.getName(), oldDate);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 177).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5561369)